server/sonar-server/src/test/java/org/sonar/server/qualitygate/QgateProjectFinderTest.java

```diff
@@ -69,17 +70,18 @@
   private ComponentDbTester componentDbTester = new ComponentDbTester(dbTester);
   private QualityGateDto qGate;
   private QgateProjectFinder underTest = new QgateProjectFinder(dbClient, userSession);
+  private OrganizationDto organization;
 
   @Before
   public void setUp() throws Exception {
-    qGate = new QualityGateDto().setName("Default Quality Gate").setUuid(Uuids.createFast());
-    dbClient.qualityGateDao().insert(dbSession, qGate);
+    organization = dbTester.organizations().insert();
```

Comment:
Next time, please remove creation of input objects in the setup, as it's very misleading as we have no idea in which tests they are used